src/Test/StateMachine.hs

```diff
@@ -47,7 +47,7 @@ sequentialProperty
      (model :: (TyFun ix * -> *) -> *)
      (m     :: * -> *)
   .  Monad m
-  => Show (Untyped' cmd ConstIntRef)
+  => Show (IntRefed cmd)
```

Comment:
We could, I left it because I thought perhaps we can do better -- if we lift `go` to the top-level analogue to `checkParallelInvariant` below?